src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        if (UserService.findByEmail(email) != (null)
+                && UserService.findByEmail(email).getPassword().equals(password)) {
+            return true;
+        }
         return false;
```

Comment:
Use the ternary operator to compress the if condition. 
```suggestion
        if (UserService.findByEmail(email) != (null)
                && UserService.findByEmail(email).getPassword().equals(password)) {
            return true;
        }
        return false;
```